### DISCUSSION TITLE
Serverless architecture

### DISCUSSION THREAD

**Discussion by spmsh** [miguelgrinberg/Flask-SocketIO#2044] [Q&A]
Hi @miguelgrinberg, referencing issue #1164, is there any update or possibility to have serverless support for flask socketIO ? Wondering if it would be possible to deploy our app on Azure Functions. Thanks!

**Comment by miguelgrinberg** [comment]
How would that work exactly? Socket.IO uses long-lived connections, which are the opposite of serverless.

You can certainly run a Socket.IO server that fires up a serverless function as a result of an event, but the server itself does not seem like a great match for this architecture.

**Comment by spmsh** [comment]
@miguelgrinberg thanks for the prompt response as always. you are right, I was being confused between Azure Functions and Azure Services. As you said Azure Functions do not support websocket so it's hardly achievable. All is fine deployed on Azure Services (except serving static files from NGINX since those are on different services/machines).
